docs/source/apps/max_sharpe_ratio_optimization.ipynb

```diff
@@ -0,0 +1,1042 @@
+{
```

Comment:
<ol><li>Nit: remove the blank cell that appears I think when you restart and run all cells</li><li>When the notebook itself looks fine, you will need to do a couple things</li><li class="ql-indent-1">in <code>docs/source/index.rst</code> add <code>apps/max_sharpe_ratio_optimization</code> to the toctree for Applications (suggest placing it in alphabetically)</li><li class="ql-indent-1"><code>cd</code> into the docs folder and run <code>make clean</code> to clear out any previously built docs and <code>make html</code> to build the html docs (you'll probably see a lot of red errors, which should be due to how some of the classes are named - that should be fine unless your docs don't build).</li><li class="ql-indent-1">open <code>docs/build/html/index.html</code> and/or <code>docs/build/html/apps/max_sharpe_ratio_optimization.html</code> (which is the doc you'll create after adding to the <code>index.rst</code> file). Now you can see how everything rendered and if anything looks off.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/417/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>